.travis.yml

```diff
@@ -1,11 +1,16 @@
+os: linux
 language: node_js
 node_js:
 - '12'
 addons:
   chrome: stable
-before_install:
-- yarn install
-script: "./scripts/travis.sh"
+install: yarn install
+before_script: "source scripts/helpers.sh"
+script:
+  - "fold yarn_check_links travis_wait 30 yarn check-links"
```

Comment:
Adding `travis_wait 30` here will hopefully solve our issue of timeouts on building the site (happens within `yarn check-links`)